grpc.go

```diff
@@ -359,6 +378,18 @@ func (a *agentGRPC) CreateContainer(ctx context.Context, req *pb.CreateContainer
 		return emptyResp, err
 	}
 
+	ctr := &container{
+		id:        req.ContainerId,
+		processes: make(map[string]*process),
+		mounts:    mountList,
+	}
+
+	a.sandbox.setContainer(req.ContainerId, ctr)
+
+	// Make sure we rollback the mounts performed in case the container
```

Comment:
it's not just the mounts - it is 'anything that needs undoing' (and we do a touch more cleanup there I think).

Reply:
let me change this

Reply:
done